src/framework/mpe/soundid.h

```diff
@@ -458,6 +459,7 @@ inline const std::unordered_map<SoundId, String> ID_STRINGS
     { SoundId::Erhu, String(u"erhu") },
     { SoundId::Contrabass, String(u"contrabass") },
     { SoundId::Octobass, String(u"octobass") },
+    { SoundId::TimpleCanario, String(u"timple") },
```

Comment:
Can you rename the SoundId to Timple, or change the string to “timple_canario”?

Reply:
Done. Thanks for your review.